src/SprykerEco/Zed/Payone/Persistence/Propel/Schema/spy_payone.schema.xml

```diff
@@ -81,6 +81,9 @@
 
         <column name="work_order_id" type="VARCHAR" size="30"/>
 
+        <column name="pay_method" type="VARCHAR" size="30"/>
```

Comment:
* This change will trigger a major, are you sure about it ?

Reply:
I have not found another solution for sharing data between different checkout steps